examples/examples_config_gen/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"encoding/hex"
+	"flag"
+	"fmt"
+
+	ec "github.com/bsv-blockchain/go-sdk/primitives/ec"
+	"github.com/bsv-blockchain/go-wallet-toolbox/examples/internal/example_setup"
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/defs"
+	"github.com/bsv-blockchain/go-wallet-toolbox/examples/internal/show"
```

Comment:
Imports are not sorted

```suggestion
	ec "github.com/bsv-blockchain/go-sdk/primitives/ec"
	"github.com/bsv-blockchain/go-wallet-toolbox/examples/internal/example_setup"
	"github.com/bsv-blockchain/go-wallet-toolbox/examples/internal/show"
	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/defs"
```

Apparently our lint formatter has no power in the `examples` directory